modules/installation-guide/examples/system-variables.adoc

```diff
@@ -102,13 +102,11 @@ pass:[<!-- vale off -->]
  `+CHE_INFRA_KUBERNETES_SINGLEHOST_GATEWAY_CONFIGMAP__LABELS+`,"`+app=che,component=che-gateway-config+`","Defines labels which will be set to ConfigMaps configuring single-host gateway." 
  `+CHE_INFRA_KUBERNETES_INGRESS_DOMAIN+`,"","Used to generate domain for a server in a workspace in case property `che.infra.kubernetes.server_strategy` is set to `multi-host`" 
- `+CHE_INFRA_KUBERNETES_NAMESPACE+`,"","DEPRECATED - Do not change the value of this property otherwise the existing workspaces will loose data. Do not set it on new installations. Defines Kubernetes namespace in which all workspaces will be created. If not set, every workspace will be created in a new namespace, where namespace = workspace id It's possible to use `<username>` and `<userid>` placeholders (for example: `che-workspace-<username>`). In that case, new namespace will be created for each user. Service account with permission to create new namespace must be used. Ignored for OpenShift infra. Use `che.infra.openshift.project` instead If the namespace pointed to by this property exists, it will be used for all workspaces. If it does not exist, the namespace specified by `che.infra.kubernetes.namespace.default` will be created and used. This parameter is subject to removal in future releases. Legacy workspaces located in this namespace may become unreachable in future releases. See the documentation about possible next steps." 
  `+CHE_INFRA_KUBERNETES_NAMESPACE_CREATION__ALLOWED+`,"`+true+`","Indicates whether {prod-short} server is allowed to create {orch-namespace} for user workspaces, or they're intended to be created manually by cluster administrator. This property is also used by the OpenShift infra." 
- `+CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT+`,"`+<username>-che+`","Defines Kubernetes default namespace in which user's workspaces are created if user does not override it. It's possible to use `<username>` and `<userid>` placeholders (for example: `che-workspace-<username>`). In that case, new namespace will be created for each user (or workspace). Used by OpenShift infra as well to specify a Project. The `<workspaceid>` placeholder is deprecated and is subject to removal in future releases. The `<username>` or `<userid>` placeholder is mandatory." 
+ `+CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT+`,"`+<username>-che+`","Defines Kubernetes default namespace in which user's workspaces are created if user does not override it. It's possible to use `<username>` and `<userid>` placeholders (for example: `che-workspace-<username>`). In that case, new namespace will be created for each user. Used by OpenShift infra as well to specify a Project. The `<username>` or `<userid>` placeholder is mandatory." 
```

Comment:
I suppose it's generated based on che.properties, but still, I'm not sure if

> if user does not override it.

makes sense anymore

Reply:
Then we need to address it in `che.properties`.
Can you please create an issue?